mbuild/formats/lammpsdata.py

```diff
@@ -41,6 +41,7 @@ def write_lammpsdata(
     use_urey_bradleys=False,
     use_rb_torsions=True,
     use_dihedrals=False,
+    borrowed_charmm=False,
```

Comment:
Can we come up with a different name for this flag? Maybe `ignore_charmm_weighting` or something?

Reply:
Yes, I was wondering what I should call it.  I didn't really have good ideas for it at the time.